services/app/src/components/gameFooter/GameFooterButtons.jsx

```diff
@@ -19,16 +19,16 @@ const GameFooterButtons = ({ skipCurrentPlayerMove }) => {
       </button>
 
       
-        <button 
-          type="button"
-          className="skip-btn padding-x-m padding-y-s rubik-font" 
-          disabled={finished || isCurrentPlayerSpectator} 
-          onClick={() => skipCurrentPlayerMove()}
-        >
-          <ExplanatoryTooltip title="Skips your move. The game continues" placement="right">
-          Skip Move
-          </ExplanatoryTooltip>
-        </button>
+      <button 
+        type="button"
+        className="skip-btn font-size-s padding-x-m padding-y-s rubik-font"
```

Comment:
why was this added?

Reply:
It came from main branch during the merge.
In the main branch: ```className=`"skip-btn font-size-s padding-x-m padding-y-s rubik-font" ```
Don't know why its highlighted as changed, but this line is the same as in main branch

Reply:
Why is this class needed in the first place?